students/mattmaeda/session07/html_render.py

```diff
@@ -0,0 +1,173 @@
+#!/usr/bin/env python3
+# coding=utf-8
+from __future__ import unicode_literals
+
+class Element(object):
+    """ class for rendering HTML element
+    """
+    def __init__(self, content=None, **kwargs):
+        self.tag = "tag"
+        self.elements = []
+        self.kwargs = {}
+
+        if content is not None:
+            self.append(content)
+
+        if kwargs is not None:
+            self.kwargs = kwargs
+
+    def append(self, content):
+        """ Appends content to class attribute elements list
+        """
+        self.elements.append(content)
+
+
+    def render(self, file_out, ind="", depth=1):
```

Comment:
if you are passing in depth, then no need to pass in indentation also....

the size of indentation should be a class attribute, and the depth is how many to use.